cranelift/filetests/filetests/runtests/i128-bricmp.clif

```diff
@@ -232,17 +232,3 @@ block2:
 ; run: %i128_bricmp_uge(0xFFFFFFFF_FFFFFFFF_FFFFFFFF_FFFFFFFD, 0xFFFFFFFF_FFFFFFFF_FFFFFFFF_FFFFFFFF) == false
 ; run: %i128_bricmp_uge(0xC0FFEEEE_C0FFEEEE_00000000_00000000, 0xDECAFFFF_DECAFFFF_00000000_00000000) == false
 ; run: %i128_bricmp_uge(0xDECAFFFF_DECAFFFF_00000000_00000000, 0xC0FFEEEE_C0FFEEEE_00000000_00000000) == true
-
-function %i128_bricmp_of(i128, i128) -> b1 {
```

Comment:
Any particular reason for this test to be deleted?

Reply:
Yes, s390x crashes when compiling it.

It didn't so far because it didn't have a `; run` declaration so we never compiled it.

It also looks like a artifact from when it was moved to [i128-bricmp-overflow.clif](https://github.com/bytecodealliance/wasmtime/blob/main/cranelift/filetests/filetests/runtests/i128-bricmp-overflow.clif)

Reply:
Ah, I see, thanks; this seems reasonable then.